packages/studio-ui/src/web/views/FlowBuilder/nodeProps/ActionModalForm.tsx

```diff
@@ -8,15 +10,34 @@ import ContentPickerWidget from '~/components/Content/Select/Widget'
 import { LinkDocumentationProvider } from '~/components/Util/DocumentationProvider'
 import { RootReducer } from '~/reducers'
 
-import ParametersTable from './ParametersTable'
+import ParametersTable, { Parameter } from './ParametersTable'
 import SelectActionDropdown from './SelectActionDropdown'
 import style from './style.scss'
 
+interface ActionMetadata {
+  params: ActionParameterDefinition[]
+  title: string
+  description: string
+  category: string
+}
```

Comment:
I think this interface already exist in studio-be, I suggest moving it to `common/typings` and using the same interface for backend and frontend (less prone to errors)

Reply:
Good catch thanks!!